conans/model/ref.py

```diff
@@ -19,10 +19,10 @@ def validate_conan_name(name):
                           " at least %s characters." % (name, ConanFileReference.min_chars)
             else:
                 message = "'%s' is an invalid name. Valid names MUST begin with a "\
-                            "letter or number, have between %s-%s chars, including "\
```

Comment:
Typically I try to separate formatting/style/renames, etc. into different PRs, so that new functionality and behavior is not cluttered with not relevant changes. I know it is annoying to see bad formatted code, but it makes reviewers work easier. Thanks in any case for cleaning things up! :)